src/ucp/wireup/select.c

```diff
@@ -534,8 +534,8 @@ static double ucp_wireup_rndv_score_func(const uct_md_attr_t *md_attr,
                                          const uct_iface_attr_t *iface_attr,
                                          const ucp_address_iface_attr_t *remote_iface_attr)
 {
-    /* highest bandwidth */
-    return iface_attr->bandwidth;
+    /* highest bandwidth with lowest overhead */
+    return (iface_attr->bandwidth) * (1 / iface_attr->overhead);
```

Comment:
using a message size of 256KB doesnt change the thresholds and chooses the correct tl so it's ok.
the purpose of this commit is to make the choice of the transport for rndv more accurate, for example, make ucx choose rc_x instead of rc.